docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -321,8 +321,8 @@ These guidelines are the policy for inclusion in the Ansible package and are in
 licensing and legal concerns that may otherwise affect your code.
 
 
-Contributor License Agreements
-==============================
+Contributor License Agreements (CLAs) and the DCO
```

Comment:
```suggestion
Contributor License Agreement (CLA) and the DCO
```

Reply:
My suggested edit above is only if we leave this here... it gets around the false spelling error CI failure.

Reply:
I've removed CLAs for now so we can park this until it returns (if ever)